### ISSUE TITLE
Updating button language for image widgets

### ISSUE THREAD

**Issue by alyblenkin** [getodk/collect#6041]
We use a few words for the image widgets (e.g., picture, image). 

We want to be consistent so that when we have two buttons side by side, such as "Take picture" and "Choose image," we use the same word. 

- "Photo" is very common with other apps when choosing from a library or taking a photo
- Alternatively, we use "Image" so it matches the name of the widget 

I don't think this is a high priority because it's been like and hasn't caused any confusion, but I wanted to capture it.

**Comment by lognaturel** [comment]
When you select from gallery, you may select an image that is not a photo. I imagine that's not super common but it's definitely possible and we know it happens with annotations. Maybe it's a bit pedantic but for someone who's selecting a schematic it could be weird to tap "choose photo!"

**Comment by alyblenkin** [comment]
> When you select from gallery, you may select an image that is not a photo. I imagine that's not super common but it's definitely possible and we know it happens with annotations. Maybe it's a bit pedantic but for someone who's selecting a schematic it could be weird to tap "choose photo!"

True! Another alternative is "Choose file" but I think "Choose image" works perfectly fine too.
